src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -64,13 +65,13 @@ class RoutePathView extends React.Component<IRoutePathViewProps, IRoutePathViewS
             <div className={s.routePathView}>
                 <div className={s.flexInnerRow}>
                     <RoutePathViewTabButtons
-                        tabs={['Reitinsuunta', 'Solmut ja linjat']}
+                        tabs={[RoutePathViewTabType.ROUTE, RoutePathViewTabType.NODES_AND_LINES]}
```

Comment:
```
const tabs = [{tab: RoutePathViewTabType.Route, content: <RoutePathViewTab routePath={this.state.routePath}}, {tab:..., content:...}]
```
Could be better to have this list dynamic for both tabs generated and view shown.

Reply:
Or note this as a todo if there's a hurry to get this before demo